hq/app/logic/DateUtils.scala

```diff
@@ -12,7 +12,9 @@ object DateUtils {
   }
   def toISOString(dateTime: DateTime) = ISODateTimeFormat.dateTime().print(dateTime)
 
-  def dayDiff(date: Option[DateTime]): Option[Long] =  date.map(new Duration(_, DateTime.now(DateTimeZone.UTC)).getStandardDays)
+  def dayDiff(date: Option[DateTime]): Option[Long] =  date.map(dayDiff)
+
+  def dayDiff(date: DateTime): Long =  new Duration(date, DateTime.now(DateTimeZone.UTC)).getStandardDays
```

Comment:
@philmcmahon is method overloading like recommendable in scala or best to avoid?

Reply:
looks good to me!